tools/system_libs.py

```diff
@@ -594,8 +595,10 @@ class Dummy(object):
     else:
       raise Exception('did not find libc?')
 
-  # Add libc-extras at the end, as libc may end up requiring them, and they depend on nothing.
-  system_libs += [('libc-extras', ext, create_libc_extras, libc_extras_symbols, ['libc_extras'], False)]
+  # Add libc-extras and compiler-rt at the end, as libc may end up requiring
+  # them, and they depend on nothing.
+  system_libs += [('libc-extras', ext, create_libc_extras, libc_extras_symbols, [], False)]
+  system_libs += [('compiler-rt', 'a', create_compiler_rt, compiler_rt_symbols, [], False)]
```

Comment:
Talked in-person, compiler-rt and libc have circular dependencies

Reply:
What is the dependency of compiler-rt on libc? (also, that fact invalidates the comment before these two lines)